code-challenges.js

```diff
@@ -39,28 +55,60 @@ var myNumbers2 = [8, -7, 0, 6, 2]
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
-
-
+const noVowels = (string) => {
+  let newArray = string.split("")
+  let filteredArray = newArray.filter(value => {
+    return value !== "a" && value !== "e" && value !== "i" && value !== "o" && value !== "u"
+  })
+  let newString = filteredArray.join("")
+  return newString
+}
+console.log(noVowels(stringWithVowels2))
 
 
 // --------------------5) Copy/paste your code from #4. Refactor your code to include non-string edge cases. Inform your user that the variable passed into the vowel removal function is not a string.
 // Use the test variables provided below. Expected output: "true is not a string" "42 is not a string"
 
 var notAString1 = true
 var notAString2 = 42
-
-
-
+//
+// const noVowels = (string) => {
+//   // let newArray = string.split("")
+//   // let filteredArray = newArray.filter(value => {
+//   //   return value !== "a" && value !== "e" && value !== "i" && value !== "o" && value !== "u"
+//   // })
+//   // let newString = filteredArray.join("")
+//   // return newString
+//   if(true !== string){
+//     return "true is not a string"
+//   }
+//   else if(42 !== string) {
+//     return "42 is not a string"
+//   }
+// }
```

Comment:
I think the problem here is that your return is ending your function early on, it needs to be the VERY last thing to happen in your function.